loop.cpp

```diff
@@ -0,0 +1,31 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+/**              
+ * main - uses strdup to create a new string, loops forever-ever
+ *                
+ * Return: EXIT_FAILURE if malloc failed. Other never returns
+ */
+int main(void)
+{
+     char *s;
+     unsigned long int i;
+     char t[15]  = "Holberton";
+     int temp = 1234;
+     s = strdup("Meg");
+     if (s == NULL)
+     {
+          fprintf(stderr, "Can't allocate mem with malloc\n");
+          return (EXIT_FAILURE);
+     }
+     i = 0;
+     while (s)
+     {
+          printf("[%lu] %s (%p)\n", i, s, (void *)s);
+          sleep(3);
+          i++;
+     }
+     return (EXIT_SUCCESS);
+}
```

Comment:
empty line to be added

Reply:
This is the code you intend to use for testing right?
If so then I recommend you add a separate folder for adding tests.

Reply:
Oh, right! I'll do that.